.github/workflows/CI.yml

```diff
@@ -124,6 +144,21 @@ jobs:
             ./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="docker-cluster"
           fi
 
+      - name: Mixed cluster backwards compatibility tests
+        run: |
+          echo "Running mixed cluster backwards compatibility tests ..."
+          ./gradlew adBwcCluster#mixedClusterTask -DmixedCluster=123 -Dtests.security.manager=false
+
+      - name: Rolling upgrade backwards compatibility tests
```

Comment:
This is nice to separate out dedicated tests but is there a way to run all BWC tests with a single command?

Reply:
There is currently no task to run that, I can add one that runs all three of them together.

Reply:
Yeah definitely, I think its worth adding that task and will help us when we would want to publish nightly builds and run these tests in one go.

Reply:
Definitely in agreement, I will work on adding a single task which can be invoked to run the bwc tests after this PR.